src/Output/LogOutputBuffer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Output;
+
+final readonly class LogOutputBuffer implements OutputBuffer
```

Comment:
Implementation for background tasks where output should be written to a log file instead of STDOUT